exercises/robot-simulator/config/checkstyle/checkstyle.xml

```diff
@@ -0,0 +1,369 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!DOCTYPE module PUBLIC
+        "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+        "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
+
+<!-- This is a checkstyle configuration file. For descriptions of
+what the following rules do, please see the checkstyle configuration
+page at http://checkstyle.sourceforge.net/config.html -->
+
+<module name="Checker">
+
+    <module name="FileTabCharacter">
```

Comment:
Is it possible to add a rule to check that files are indented correctly with four spaces? 🙂

Reply:
I'll give a look to rule syntax, but I imagine that it's possible

Reply:
done